threatfox/package.yaml

```diff
@@ -158,12 +157,13 @@ examples:
       from "https://threatfox-api.abuse.ch/api/v1/", data = {
         query: "get_iocs",
         days: 7
-      }
+      } { read_json }
```

Comment:
NB: I find the two blocks next to each other really hard to grok.

Reply:
I think it's fine to merge this, it's an improvement even with this in.